SECP-service/src/main/resources/assets/app/js/register/register-controller.js

```diff
@@ -1,3 +1,21 @@
 angular.module('SECP')
-    .controller('RegisterController', ['$scope', function ($scope) {
-    }]);
\ No newline at end of file
+    .controller('RegisterController', ['$scope', 'RegisterService',
+        function ($scope, RegisterService) {
+        $scope.verifyUsernameUrl = "/SECP/verify?username=";
+        $scope.verifyEmailUrl = "/SECP/verify?email=";
+
+        $scope.register = function () {
+            RegisterService.register($scope.user).then(function(res) {
+                if(res.status == 201) {
+                    swal('Welcome to SECP!', 'Registration successful!','success');
```

Comment:
Just want to confirm, it will be the admins registering the users right?

Reply:
Yes that’s correct. This page would only be visible to admins :)

Reply:
Perfect.